tutorials/Augurpy.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    result_adata, results = calculate_auc(loaded_data, random_forest, random_state=51)
</pre><p>Let's stay with a seed of 42 :) People will appreciate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/augurpy/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>